### PR TITLE
Stop patching 'default' namespace in ACM policy (for HCP)

### PR DESCRIPTION
**Pull request by jewzaam**
### What type of PR is this?
bug

### What this PR does / why we need it?
tl;dr:  move patch to new sub-dir which is excluded from ACM Policy but still included in the hive SSS generation

Also see deploy/osd-project-request-template/nonhypershift/README.md

### Which Jira/Github issue(s) this PR fixes?
https://issues.redhat.com/browse/OSD-15883

### Special notes for your reviewer:

Related: https://issues.redhat.com/browse/OSD-15886

### Pre-checks (if applicable):
- [ ] Tested latest changes against a cluster
- [ ] Included documentation changes with PR
- [ ] If this is a new object that is not intended for the FedRAMP environment (if unsure, please reach out to team FedRAMP), please exclude it with:

    ```yaml
    matchExpressions:
    - key: api.openshift.com/fedramp
      operator: NotIn
      values: ["true"]
    ```

### CODE REVIEW

**Review by fahlmant — APPROVED** [review]
/lgtm
/approve
